com.unity.multiplayer.mlapi/Runtime/NetworkVariable/Collections/NetworkList.cs

```diff
@@ -103,7 +104,7 @@ public bool IsDirty()
                 return false;
             }
 
-            if (m_NetworkBehaviour.NetworkManager.NetworkTime - LastSyncedTime >= (1f / Settings.SendTickrate))
+            if (m_NetworkBehaviour.NetworkManager.LocalTime.FixedTime - LastSyncedTime.FixedTime >= (1f / Settings.SendTickrate))
```

Comment:
Given how often IsDirty is called, I have this urge to suggest we find a way to avoid computing `1f / Settings.SendTickrate` each time.  The compiler can't assume it won't change from call to call.  I guess that brings up a thought, which is do we expect the SendTickrate to vary?  If not, we might take advantage of that.  But maybe we want the ability to dilate time, esp on clients?

Reply:
Agreed, but this should be part of the NetworkVariable rework we do when we move to snapshots and not part of this PR.

Reply:
yeah, we've got those division all over the place between frequency and period. Maybe we need to figure the right way? Personally, I'd always only ever store durations, never rates, but I don't want to be dogmatic.